.github/workflows/release.yaml

```
@@ -14,43 +14,18 @@ on:
           - major
 
 jobs:
-  release-to-pypi:
-    uses: bakdata/ci-templates/.github/workflows/python-poetry-release.yaml@v1.18.2
+  create-github-release-push-tag:
+    runs-on: ubuntu-22.04
+    uses: bakdata/ci-templates/.github/workflows/python-poetry-release.yaml@v1.22.0
     with:
-      release-type: ${{ github.event.inputs.release-type }}
-      publish-to-test: false
-      python-version: "3.10"
+      release-type: ${{inputs.release-type}}
```

surrounding space for consistency
```suggestion
      release-type: ${{ inputs.release-type }}
```